### ISSUE TITLE
【bug】Some weights of the model checkpoint at openai/clip-vit-large-patch14 were not used when initializing CLIPTextModel

### ISSUE THREAD

**Issue by XuJianzhi** [kohya-ss/sd-scripts#273]
![image](https://user-images.githubusercontent.com/30584453/224210162-df4606b7-90ce-480a-93fc-c5cb86779ae3.png)

CUDA_VISIBLE_DEVICES=2 accelerate launch --num_cpu_threads_per_process 2 train_network.py \
    --pretrained_model_name_or_path=/use/model/stable_diffusion/v1-5-pruned.safetensors \
    --train_data_dir=/use/train_lora/sd-scripts/data/car-geely/train \
    --output_dir=/use/train_lora/sd-scripts/data/car-geely/out \
    --prior_loss_weight=1.0  \
    --resolution=683,512 \
    --train_batch_size=1 \
    --learning_rate=1e-4  \
    --max_train_steps=4400 \
    --optimizer_type=AdamW8bit  \
    --xformers  \
    --mixed_precision=fp16  \
    --save_every_n_epochs=1  \
    --save_model_as=safetensors  \
    --clip_skip=2  \
    --seed=42  \
    --color_aug  \
    --network_module=networks.lora

why？？？？？ thank you !!
